tests/testthat/test-layers.R

```diff
@@ -1,89 +1,109 @@
-  expect_s3_class(l_data[[2]], class = c("sf", "data.frame"))
-  expect_gt(nrow(l_data[[1]]), 0)
-  expect_gt(nrow(l_data[[2]]), 0)
+    skip_if_offline()
+    wfs <- create_biology_wfs()
+    l_data <- emodnet_get_layers(
+        wfs = wfs,
+        layers = c("mediseh_zostera_m_pnt", "mediseh_posidonia_nodata"))
+    l_crs <- purrr::map_int(l_data, ~sf::st_crs(.x)$epsg) %>% unique()
+
+    expect_length(l_crs, 1)
+    expect_equal(l_crs, 4326)
+    expect_type(l_data, "list")
+    expect_length(l_data, 2)
+    expect_s3_class(l_data[[1]], class = c("sf", "data.frame"))
+    expect_s3_class(l_data[[2]], class = c("sf", "data.frame"))
+    expect_gt(nrow(l_data[[1]]), 0)
+    expect_gt(nrow(l_data[[2]]), 0)
 
 })
 
-test_that("crs trasform works from server", {
-  l_data <- emodnet_get_layers(service = "seabed_habitats_individual_habitat_map_and_model_datasets",
-                               layers = "dk003070", crs = 4326)
-  l_crs <- purrr::map_int(l_data, ~sf::st_crs(.x)$epsg) %>% unique()
-  expect_length(l_crs, 1)
-  expect_equal(4326, l_crs)
+test_that("crs transform works from server", {
+    skip_if_offline()
+    wfs <- create_biology_wfs()
+    l_data <- emodnet_get_layers(
+        wfs = wfs,
+        layers = "mediseh_zostera_m_pnt",
+        crs = 4326
+    )
+    l_crs <- purrr::map_int(l_data, ~sf::st_crs(.x)$epsg) %>% unique()
+
+    expect_length(l_crs, 1)
+    expect_equal(l_crs, 4326)
 })
 
-
-
-wfs_cml <- emodnet_init_wfs_client("chemistry_marine_litter")
-layers <- c("sl_fishing", "sl_plasticbags")
-
 test_that("get layers works on wfs object", {
-  l_data <- emodnet_get_layers(wfs = wfs_cml, layers = layers)
-  l_crs <- purrr::map_int(l_data, ~sf::st_crs(.x)$epsg) %>% unique()
-
-  expect_length(l_crs, 1)
-  expect_equal(4326, l_crs)
-  expect_equal(length(l_data), 2)
-  expect_type(l_data, "list")
-  expect_s3_class(l_data[[1]], class = c("sf", "data.frame"))
-  expect_s3_class(l_data[[2]], class = c("sf", "data.frame"))
-  expect_gt(nrow(l_data[[1]]), 0)
-  expect_gt(nrow(l_data[[2]]), 0)
+    skip_if_offline()
+    wfs <- create_biology_wfs()
+    layers <- c("mediseh_zostera_m_pnt", "mediseh_posidonia_nodata")
+    l_data <- emodnet_get_layers(wfs = wfs, layers = layers)
+    l_crs <- purrr::map_int(l_data, ~sf::st_crs(.x)$epsg) %>% unique()
+
+    expect_length(l_crs, 1)
+    expect_equal(l_crs, 4326)
+    expect_type(l_data, "list")
+    expect_length(l_data, 2)
+    expect_s3_class(l_data[[1]], class = c("sf", "data.frame"))
+    expect_s3_class(l_data[[2]], class = c("sf", "data.frame"))
+    expect_gt(nrow(l_data[[1]]), 0)
+    expect_gt(nrow(l_data[[2]]), 0)
 })
 
-test_that("crs trasform works from wfs object", {
-  l_data <- emodnet_get_layers(wfs = wfs_cml, layers = layers, crs = 3857)
-  l_crs <- purrr::map_int(l_data, ~sf::st_crs(.x)$epsg) %>% unique()
-  expect_length(l_crs, 1)
-  expect_equal(3857, l_crs)
+test_that("crs transform works from wfs object", {
+    skip_if_offline()
```

Comment:
Question: Do we need to skip if offline if we are going to use cached results through httptest?